libs/service-portal/information/src/components/PersonalInformation/Forms/ProfileForm/components/Inputs/ReadOnlyWithLinks.css.ts

```diff
@@ -0,0 +1,16 @@
+import { style } from '@vanilla-extract/css'
+import { themeUtils } from '@island.is/island-ui/theme'
+
+export const linkContainer = style({
+  paddingBottom: 0,
```

Comment:
We should be able to use props on the `<Box>` component to set all of these instead of needing this style file.